addons/wasmedge/enable

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SNAP_CHANNEL=--candidate
+PATCH_DIR=`dirname "$0"`/patch
+
+echo "Enabling wasmedge via crun OCI runtime"
+
+echo "Install/Upgrade crun-wasmedge from snapstore"
+[ ! -d "/snap/crun-wasmedge/current" ] && sudo snap install $SNAP_CHANNEL crun-wasmedge || sudo snap refresh $SNAP_CHANNEL crun-wasmedge
+sudo ln -sf /snap/crun-wasmedge/current/crun.wrapper /usr/local/bin/crun
+
+echo "Switching OCI runtime to crun with wasmedge support"
+sudo patch -p1 $SNAP_DATA/args/containerd-template.toml $PATCH_DIR/0001-add-crun-to-containerd-template.toml.patch 
```

Comment:
There are two things here we need to address:

- the `patch` binary is not part of microk8s so this will fail on systems that do not have it on the host.
- applying the changes we want as a patch, although makes this code straightforward and clean, makes this addon a bit fragile. If/when the containerd configuration changes this patch will need to be re-created. Could we instead use something like `sed` to replace the `default_runtime_name = "${RUNTIME}"` with the part we have for this patch?

Reply:
Switched to `sed`.